src/api/user.test.tsx

```diff
@@ -17,3 +17,28 @@ describe('useUsers', () => {
     expect(result.current.data).toEqual(expected);
   });
 });
+
+describe('useAddUser', () => {
+  it('posts a request to add a user and returns successful response', async () => {
+    const { result } = renderHook(() => useAddUser(), {
+      wrapper: hooksWrapperWithProviders(),
+    });
+    expect(result.current.isIdle).toBe(true);
+
+    result.current.mutate({
+      ...usersJson[0],
+      _id: usersJson[0].username,
+      sha256_password: 'test',
+    });
+
+    await waitFor(() => {
+      expect(result.current.isSuccess).toBeTruthy();
+    });
+
+    expect(result.current.data).toEqual(usersJson[0].username);
+  });
+
+  it.todo(
+    'sends axios request to post user session and throws an appropriate error on failure'
+  );
```

Comment:
Do these TODOs ever get implemented? Looking at the delete PR it doesn't seem like it...

Reply:
The errors are handle in the `mutateAsync`. This shouldn't have been added in. I'll remove it